Makefile

```diff
@@ -1,20 +1,30 @@
 
 .PHONY: publish-vercel, install
 
-# None of this is needed since publish is happening
-# in the CI environment using the statics from PyPI, not from a local build
-# Need to re-evaluate how to use locally created plugins/statics for this in a future iteration. For now, just use published assets for simplicity.
+# Flags used during local dev and in the published demo.
+DATASETTE_DEMO_FLAGS := \
+    --plugins-dir=demo/demo-plugins \
+		--metadata=demo/demo-metadata.yml
+
+# We didn't need to install yarn, npm packages etc
+# in the CI environment using the statics from PyPI, not from contents of the local repo.
+# We need to re-evaluate how to use locally created plugins/statics for this someday. For now, just use published assets on PyPI for simplicity.
 
 install:
 	pip install -r demo/requirements.txt
-# 	# pip install -e .
 
 # https://github.com/simonw/datasette-publish-vercel#other-options
 publish-vercel: install
-
 	 datasette publish vercel demo/happy_planet_index.db \
   			--project=datasette-nteract-data-explorer \
 				--scope=datasette-visualization-plugin-demos \
 				--token=${VERCEL_TOKEN} \
 				--install datasette-nteract-data-explorer \
-				--public
+				--public \
+				${DATASETTE_DEMO_FLAGS}
+
+
+run-demo:
+	datasette -i demo/happy_planet_index.db \
+		${DATASETTE_DEMO_FLAGS}
+# --template-dir=demo/demo-templates
```

Comment:
Commented this out because overriding `base.html` had no effect. Only full pages or the "row" partial can be overridden.

https://docs.datasette.io/en/stable/custom_templates.html#custom-templates